coord/src/dist_aggregate.c

```diff
@@ -279,6 +279,12 @@ static void buildMRCommand(RedisModuleString **argv, int argc, int profileArgs,
   // Numeric responses are encoded as simple strings.
   tmparr = array_append(tmparr, "_NUM_SSTRING");
 
+  int dialectOffset = RMUtil_ArgIndex("DIALECT", argv + 3 + profileArgs, argc - 3 - profileArgs);
+  if (dialectOffset != -1 && dialectOffset + 3 + 1 + profileArgs < argc) {
+    tmparr = array_append(tmparr, "DIALECT");
+    tmparr = array_append(tmparr, RedisModule_StringPtrLen(argv[dialectOffset + 3 + 1 + profileArgs], NULL));  // the dialect
+  }
```

Comment:
Please explain, what this fixes? Is it an existing bug or only needed for this PR?

Reply:
`DIALECT` arg was not being passed to shards with `AGGREGATE`.
It was only being passed with `SEARCH`.

Reply:
Will move to a separate PR

Reply:
Moved to #3172